census_view.cpp

```diff
@@ -1393,12 +1443,14 @@ void CensusView::UponColumnWidthVarying(wxCommandEvent&)
 {
     autosize_columns_ = true;
 
-    wxWindowUpdateLocker u(list_window_);
-    for(int j = 0; j < bourn_cast<int>(list_window_->GetColumnCount()); ++j)
+    if(!Update())
         {
-        list_window_->GetColumn(j)->SetWidth(wxCOL_WIDTH_AUTOSIZE);
+        wxWindowUpdateLocker u(list_window_);
+        for(unsigned int j = 0; j < list_window_->GetColumnCount(); ++j)
```

Comment:
Just FYI, I've undone this (unwanted) change (as well as the similar one below) in #113.